inventory_management/inventory_management/doctype/stock_ledger_entry/stock_ledger_entry.json

```diff
@@ -0,0 +1,79 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2024-01-16 11:31:00.481115",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "section_break_6ruo",
+  "item",
+  "warehouse",
+  "qty_change",
+  "valuation_rate",
```

Comment:
`in_out_rate` field is missing. This is the actual rate at which the item enters/leaves your warehouse. Whereas Valuation Rate is related to all previous SLEs.

This is what the mapping looks like from Stock Entry -> Stock Ledger Entry

<img width="646" alt="image" src="https://github.com/vinyselopal/InventoryManagementFrappe/assets/24353136/3a879da3-ba3f-4ee9-90f3-681f2690f8e7">